components/FormikFields/PaperInputPicker/index.js

```diff
@@ -62,6 +65,34 @@ const PaperInputPicker = ({
   const [pictureUris, setPictureUris] = React.useState({});
   const [image, setImage] = React.useState(null);
 
+  const [numberedQestions, setNumberedQuestions] = React.useState({});
+  const [questionsToRepeat, setQuestionsToRepeat] = React.useState([]);
+  const [additionalQuestions, setAdditionalQuestions] = React.useState([]);
+  const [individualLoopsAdded, setIndividualLoopsAdded] = React.useState(0);
+
+  // add another loop of questions in correct position in form (additionalQuestions)
+  // update number of loops added (+1)
+  const addLoop = () => {
+    // updated formikKey to ensure uniqueness
+    let updatedQuestions = [];
+    questionsToRepeat.forEach((question) => {
+      const updatedQuestion = _.cloneDeep(question);
+      updatedQuestion.formikKey = `${updatedQuestion.formikKey}__loop${individualLoopsAdded}`;
+      updatedQuestions = updatedQuestions.concat(updatedQuestion);
+    });
+
+    setAdditionalQuestions(additionalQuestions.concat(updatedQuestions));
+    setLoopsAdded(loopsAdded + 1);
+    setIndividualLoopsAdded(individualLoopsAdded + 1);
+  };
```

Comment:
since you're just modifying states that you can pass to `Looper.js`, just move this function there

Reply:
great!